yiisoft/yii2/CVE-2018-7269.yaml

```diff
@@ -0,0 +1,8 @@
+title:     Potential SQL injection in methods `yii\db\ActiveRecord::findOne()` and `::findAll()`
+link:      http://www.yiiframework.com/news/168/releasing-yii-2-0-15-and-database-extensions-with-security-fixes/
+cve:       CVE-2018-7269
+branches:
+    2.0.x:
+        time:     2018-03-20 14:00:00
+        versions: ['<2.0.12.1', '2.0.13', '2.0.13.1', '~2.0.14.0']
+reference: composer://yiisoft/yii2
```

Comment:
same file needs to be added to yii2-dev repo.

Reply:
+